astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -29,6 +33,21 @@
 ) as dag:
     start = DummyOperator(task_id="start")
 
+    # Execute AWS command then sleep for 5 min so that cluster would be available
+    create_redshift_cluster = BashOperator(
+        task_id="create_redshift_cluster",
+        bash_command=f"aws redshift create-cluster "
+        f"--db-name {REDSHIFT_CLUSTER_DB_NAME} "
+        f"--cluster-identifier {REDSHIFT_CLUSTER_IDENTIFIER} "
+        f"--cluster-type single-node "
+        f"--node-type dc2.large  "
+        f"--master-username {REDSHIFT_CLUSTER_MASTER_USER} "
+        f"--master-user-password {REDSHIFT_CLUSTER_MASTER_PASSWORD} && sleep 4m && "
+        f"aws redshift create-cluster-snapshot "
+        f"--snapshot-identifier {REDSHIFT_CLUSTER_IDENTIFIER}-snapshot "
+        f"--cluster-identifier {REDSHIFT_CLUSTER_IDENTIFIER} && sleep 5m",
```

Comment:
Rather than an explicit sleep, we can use cluster status from the boto3 client.

Reply:
make sense. Changed it to use boto3 api and wait till it available.